firmware/common2015/utils/rtos-mgmt/helper-funcs.hpp

```diff
@@ -0,0 +1,8 @@
+#pragma once
+
+#include <cstdint>
+#include <string>
+
+void setISRPriorities(void);
```

Comment:
These functions could probably use a couple of comments :)

Also, will the ISR priorities be different for the base station and the robot?  If so, we should probably move this out of common2015 or make it more general to handle both.